cmd/harmony/main.go

```diff
@@ -504,6 +509,19 @@ func main() {
 	if *enableMemProfiling {
 		memprofiling.GetMemProfiling().Start()
 	}
+
+	if *logP2P {
+		f, err := os.OpenFile(path.Join(*logFolder, "libp2p.log"), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to open libp2p.log. %v\n", err)
+		} else {
+			defer f.Close()
+			backend1 := gologging.NewLogBackend(f, "", 0)
```

Comment:
why is there 1 at the end at var name?

Reply:
you can set multiple backend though for gologging.